cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -90,6 +90,9 @@
 
     protected static final Logger log = LogManager.getLogger(AbstractModel.class.getName());
 
+    public static final String STRIMZI_DNS_DOMAIN =
```

Comment:
Just a nit ... but `STRIMZI_DNS_DOMAIN` suggests its something of our own. In kubeadm the option is named `service-dns-domain`. What about calling it `KUBERNETES_SERVICE_DNS_DOMAIN`? Might better explain the purpose of this. Might be also worth adding some comment to explain why we have it for the years to come :-).

Reply:
I had same doubt but then decided to use the same `STRIMZI` prefix. Btw I'm happy to change, I'll do that.